twatbot.py

```diff
@@ -48,17 +48,18 @@ def sanitized_twat(twat, args=None):
 
 def build_searchbox(vars):
 	link = make_index_link(vars, exclude=['search', 'find'])
-	fill = vars['search'] if 'search' in vars else 'foo "bar baz" -quux'
+	fill, reset = (vars['search'], 'clear') if 'search' in vars else ('foo "bar baz" -quux', 'hidden')
```

Comment:
can we be a bit less smart here and use an if statement instead ?
i kinda like if i can understand what a line does with a single glance...